week-3/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,23 @@
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach(function (nums) {
+  if (nums === 0) {
+    console.log(nums);
+  }
+  else if (nums % 3 === 0 && nums % 5 === 0) {
+    console.log("FizzBuzz");
+  }
+  else if (nums % 5 === 0) {
+    console.log("Buzz");
+  }
+  else if (nums % 3 === 0) {
+    console.log("Fizz");
+  }
+  else {
+    console.log(nums);
+  }
```

Comment:
Well done! 😄 FizzBuzz is a famous way of testing your programming skill!

Reply:
thanks